Foundation/testsuite/src/ChannelTest.cpp

```diff
@@ -70,13 +70,15 @@ void ChannelTest::testSplitter()
 void ChannelTest::testAsync()
 {
 	AutoPtr<TestChannel> pChannel = new TestChannel;
-	AutoPtr<AsyncChannel> pAsync = new AsyncChannel(pChannel);
+#ifndef POCO_OS_FAMILY_WINDOWS
+	AutoPtr<AsyncChannel> pAsync = new AsyncChannel(pChannel); // FIXME It is crashing here on Windows!
```

Comment:
@Bjoe can you create an issue for this, with more details; I haven't seen it, may be something appveyor related

Reply:
@aleks-f As zosrothko said, but it is also reproductible on the "Azure" windows.

@zosrothko will you create an issue for that? I think it is a bug in `AsyncChannel`, or?

Reply:
This unittest is working well with poco-1.9.1.. so the problem should be elsewhere. I will work on it sooner or later